src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/EditorSettingsManager.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.Razor.ProjectSystem
+{
+    public abstract class EditorSettingsManager : ILanguageService
+    {
+        public abstract event EventHandler<EditorSettingsChangedEventArgs> Changed;
+
+        public abstract EditorSettings Settings { get; }
```

Comment:
It might be nicer to call this `Current` or `CurrentSettings`

Reply:
I like `Current`